spec/current_ability_spec.rb

```diff
@@ -154,6 +154,24 @@
       expect(user.current_ability.can?(:action, :subject)).to eq true
       expect(ability_proc).not_to receive(:call)
       expect(user.current_ability.can?(:action, :subject)).to eq true
+      expect(user.current_ability.cannot?(:action, :subject)).to eq false
```

Comment:
This shows that the cached `can?` and `cannot?` values are opposite of each other.